.github/workflows/integration.yml

```diff
@@ -58,8 +58,8 @@ jobs:
       repo_url: ${{ fromJson(steps.pr-api.outputs.data).head.repo.html_url }}
 
   integration:
-    needs: pr-info
-    if: ${{ needs.pr-info.outputs.allowed_user == 'true' && !contains(github.event.pull_request.labels.*.name, 'control/skip-ci') }}
+    # needs: pr-info
```

Comment:
The workflow changes in PR will not be available in this PR.

Reply:
Would I be able to get the `admin` permission? cc @cgwalters

Reply:
Oh, sorry, should be `admin` or `write` permission.

Reply:
@lukewarmtemp I previously sent you an invite to the org.  Just sent you a new one.  Can you check your spam folder?

Reply:
Got it, thanks!